sdk/storage/Azure.ResourceManager.Storage/tests/Tests/BlobContainerTests.cs

```diff
@@ -24,7 +24,7 @@ public BlobContainerTests(bool async) : base(async)
         public async Task createStorageAccountAndGetBlobContainerContainer()
         {
             _resourceGroup = await CreateResourceGroupAsync();
-            string accountName = Recording.GenerateAssetName("storage");
+            string accountName =await CreateValidAccountNameAsync("teststoragemgmt");
```

Comment:
one space after the equality sign

Reply:
and consider to define a const variable which have the prefix "teststoragemgmt" since it has been used in so many tests